test_scripts/Polices/appID_Management/ATF_P_TC_OnAppPermissionChanged_appRevoked_true.lua

```diff
@@ -0,0 +1,70 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+--     [ChangeRegistration]: appRevoked:true
+-- Description:
+-- <appID> application is assigned with specific policies (for example, appID = 123abc->
+-- "app_policies" -> "123abc" : { <specific policies> }
+-- Steps:
+-- 1. any PolicyTableUpdate trigger happens
+-- 2. PTU is valid -> <appID> gets "null" policy
+-- Expected:
+-- SDL -> HMI: OnAppPermissionChanged (<appID>, appRevoked=true, params)
+--
+-- Test:
+--     1. Preconditions:
+--        Start SDL
+--
+--     2. Steps
+--        Register application with policy id 123abc
+--        Perform PTU with structure "123abc": null
+--
+--     3. Expected result:
+--        SDL -> HMI: OnAppPermissionChanged (<appID>, appRevoked=true, params)
+--
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+  config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+  local Common = require("user_modules/shared_testcases/testCasesForPolicyAppIdManagament")
+
+--[[ General Settings for configuration ]]
+  Test = require("connecttest")
+  local mobile_session = require("mobile_session")
+
+  function Test:Pre_StartNewSession()
+    self.mobileSession2 = mobile_session.MobileSession(self, self.mobileConnection)
+    self.mobileSession2:StartService(7)
+  end
+
+--[[ Test ]]
+  function Test:RegisterNewApp_PerformPTU_Check_OnAppPermissionChanged()
```

Comment:
Add above line 
commonFunctions:newTestCasesGroup ("Test")
We need this before start of any group e.g. Preconditons, Test, Postconditions